### PR TITLE
Case activity prefactor

### PR DESCRIPTION
**Pull request by emord**
This PR was getting a little unwieldy, so breaking it up into 2 (possibly 3 PRs). This PR shouldn't have any functional changes, except for turning off sorting on the proportion row.

:fish:

cc: @NoahCarnahan

### CODE REVIEW

File: corehq/apps/reports/standard/monitoring.py [hunk 1]
```diff
@@ -228,11 +228,13 @@ def headers(self):
 
         def make_column(title, help_text, num_days):
             return DataTablesColumn(title, sort_type=DTSortType.NUMERIC,
-                                    help_text=help_text.format(num_days))
+                                    help_text=help_text.format(num_days),
+                                    sortable=False if title=="Proportion" else True)
```
Comment (dimagimon) [inline corehq/apps/reports/standard/monitoring.py]: E225 missing whitespace around operator

File: corehq/apps/es/aggregations.py [hunk 2]
```diff
@@ -102,7 +102,7 @@ def buckets_dict(self):
 
     @property
     def buckets_list(self):
-        return {Bucket(b, self._aggregations) for b in self.normalized_buckets}
+        return [Bucket(b, self._aggregations) for b in self.normalized_buckets]
```
Comment (emord) [inline corehq/apps/es/aggregations.py]: @snopoke any reason you used a set originally? I added sorting to the aggregation and using a set here doesn't preserve the order.

**Comment by NoahCarnahan** [comment]
Looks good. Test failure looks real though.

File: corehq/apps/es/aggregations.py (hunk 2 above)
Comment (NoahCarnahan) [inline corehq/apps/es/aggregations.py]: Do you want a response here before merging, or is the PR gtg?

Comment (emord) [inline corehq/apps/es/aggregations.py]: I think it should be good to merge. I did a quick look for where this was used and think that failed test was the only change needed.

**Pull request merged**